rails/app/models/media_link.rb

```diff
@@ -0,0 +1,8 @@
+class MediaLink < ApplicationRecord
+    belongs_to :story
+
+    def self.update_url(url)
+        url["watch?v="]="embed/"
+        # "watch?v=" ==> "embed/"
```

Comment:
woah. I think i'd want a test for this, as i'm not quite sure that behavior is what i expect

Reply:
@FeminismIsAwesome I don't think this was working anyways, I'll just remove it for now and we'll continue to work on it.